### DISCUSSION TITLE
Docs: Vitest

### DISCUSSION THREAD

**Discussion by ThiagoGMota** [vercel/next.js#74501] [Ideas]
### What is the documentation issue?

There is no official tutorial to install vitest in nextjs

### Is there any context that might help us understand?

Create a simple tutorial for the beginners programmers learn how to install correctly the vitest

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/testing/vitest

**Comment by drumst0ck** [comment]
In the docs link, you have a manual setup guide. What do you need?
